linter/whitelists/de.txt

```diff
@@ -468,3 +468,17 @@ Weiterlesen
 Webanwendungen
 Plenta
 Jobbörse
+Sineos
+SINEOS
```

Comment:
This should be added to the `default.txt`, because "Sineos" and "SINEOS" are not language specific.